### ISSUE TITLE
CSS Custom Props (css variables) syntax highlighting

### ISSUE THREAD

**Issue by chiangs** [sdras/night-owl-vscode-theme#217]
I would like to request a little more distinction in the syntax highlighting for css variables.
![code](https://user-images.githubusercontent.com/10984593/60586749-028c1900-9d93-11e9-97b4-d9e9668c53ad.png)

**Comment by sdras** [comment]
Definitely! This should be more legible. I'm happy to work on this but I'm low on time, so if I do it, it won't be for a couple of months. Open to PRs.

**Comment by chiangs** [comment]
I've been searching for a good reference on how to select these in the settings, but I am struggling to figure it out using TextMates' documentation. Can you share the source you referenced in building this theme? Thanks!

**Comment by chiangs** [comment]
@sdras I've got it working now, here is the regular dark theme, can you comment before i move on to the other themes?

I've opted for the purple color you used and also set it to italic when actually being used and non-italic when just being declared.

Also could you tag this as assigned to me and in progress?

![Screen Shot 2019-10-02 at 15 39 49](https://user-images.githubusercontent.com/10984593/66049017-e8e1a580-e52a-11e9-87e7-a1b72abd87d2.png)

**Comment by Arnau-Ninerola** [comment]
@chiangs That looks really good, maybe I like the second option more. Are you working on fixing this issue? If not I would't mind taking a look.

**Comment by chiangs** [comment]
@Arnau-Ninerola Yes I am actively working on it, I just wanted @sdras to have a look first and assign to me. I think I like the second one better as well.

**Comment by chiangs** [comment]
@sdras PR opened
